src/Umbraco.Infrastructure/Persistence/UmbracoDatabaseFactory.cs

```diff
@@ -248,9 +243,6 @@ private SqlContext Initialize()
                 {
                     cfg.WithInterceptor(interceptor);
                 }
-
-                cfg.WithInterceptor(new ProfiledDbConnectionInterceptor());
```

Comment:
Miniprofiler isn't really provider specific, but the order is important and this is a easy way to handle the ordering.
i.e. profiler should be before retry policy.